README.rst

```diff
@@ -258,6 +257,19 @@ and using the "file" portion as the filename of the database.
 This has the effect of four slashes being present for an absolute
 file path: sqlite:////full/path/to/your/database/file.sqlite.
 
+Nested lists
+------------
+
+Some settings such as Django's ``ADMINS`` make use of nested lists. You can use something like this to handle similar cases.
+
+.. code-block::
+
+    DJANGO_ADMINS=John:john@admin.com,Jane:jane@admin.com
+    
+.. code-block:: python
+
+    ADMINS = [x.split(':') for x in env.list('DJANGO_ADMINS')] 
```

Comment:
This LG, specific to `DJANGO_ADMINS` where it need emails, here is how I handle it. Just my 2 cents. 

```python
from email.utils import getaddresses

# In the format 'Full Name <email@example.com>, Full Name <anotheremail@example.com>'
# e.g. DJANGO_ADMINS=Full Name <email-with-name@example.com>,anotheremailwithoutname@example.com
ADMINS = getaddresses([env('DJANGO_ADMINS')])
```

https://github.com/Fueled/django-init/blob/master/%7B%7Bcookiecutter.github_repository%7D%7D/settings/production.py#L34

Reply:
Thanks, I now prefer the  `getaddresses`approach too.

I can't think of any use case where the tip I added would add enough value to justify documenting then.